tests/syscall/socket/host/CMakeLists.txt

```diff
@@ -6,7 +6,10 @@ set (EDL_FILE ../socket_test.edl)
 add_custom_command(
     OUTPUT socket_test_u.h socket_test_u.c
     DEPENDS ${EDL_FILE} edger8r
-    COMMAND edger8r --untrusted ${EDL_FILE} --search-path ${CMAKE_CURRENT_SOURCE_DIR}
+    COMMAND edger8r --untrusted ${EDL_FILE}
+               --search-path ${PROJECT_SOURCE_DIR}/edl
+               --search-path ${PROJECT_SOURCE_DIR}/edl/sgx
```

Comment:
Same question about tests... how does this sgx search path affect running the tests for optee?

Reply:
Fixed this for tests. Still looking into how I can differentiate in the samples. If they don't already run on OP-TEE I would probably defer to a PR that fixes those problems first.